README.md

```diff
@@ -30,6 +30,21 @@ Standalone agent with defaults:
 
     echo include puppet | puppet --modulepath /path_to/extracted_tarball
 
+# Integration with foreman
+
+With the 3.0.0 release the foreman integration became optional. Note that you
+need the [puppet-foreman module](https://github.com/theforeman/puppet-foreman)
+installed.
+
+It will still by default install the foreman integration when server is true,
```

Comment:
You mean literal with backticks? Isn't that a github flavor instead of plain markdown?